docs/samples/browser-plugin-meetings/index.html

```diff
@@ -48,10 +48,14 @@ <h2 class="collapsible">Meeting Management</h2>
 
               <form id="credentials">
                 <p class="note">
-                  NOTE: Click <a href="https://developer.webex.com/docs/getting-started" target="_blank">here</a> for access token
+                  NOTE: Click <a href="https://developer.webex.com/docs/getting-started" target="_blank">here</a> for
+                  access token
                 </p>
                 <div>
-                  <input id="access-token" name="accessToken" placeholder="Access Token" value="" type="text" required>
+                  <input id="guest-name" name="guestName" placeholder="Guest Name" value="" type="text">
+                  <button id="get-guest-token" class="btn-code" type="button"
+                    onclick=getGuestAccessToken()>generateGuestToken [prod only]</button>
```

Comment:
why "generateGuestToken" and not "Generate guest token"? is generateGuestToken a name of a public SDK function?

Reply:
no its just a function in app.js so added it there e

Reply:
will change it